app/components/live-participants.hbs

```diff
@@ -0,0 +1,39 @@
+<div data-test-sidebar-body-role class='body__role-user'>
```

Comment:
not-eq function is being used to compare the @minimumParticipants variable and the @role variable. This is incorrect, because the not-eq function is used to compare two values and return true if they are not equal. In this case, the @minimumParticipants variable and the @role variable are both strings, so they cannot be compared using the not-eq function.